static/app/components/group/groupSummary.tsx

```diff
@@ -248,17 +251,34 @@ export function GroupSummary({
                       <Placeholder height="1.5rem" />
                     </CardContent>
                   ) : (
-                    card.insight && (
-                      <CardContent
-                        dangerouslySetInnerHTML={{
-                          __html: marked(
-                            preview
-                              ? card.insight.replace(/\*\*/g, '') ?? ''
-                              : card.insight ?? ''
-                          ),
-                        }}
-                      />
-                    )
+                    <CardContent>
+                      {card.insightElement ? (
+                        <Fragment>
+                          {card.insightElement}
+                          {card.insight && (
+                            <div
+                              dangerouslySetInnerHTML={{
+                                __html: marked(
+                                  preview
+                                    ? card.insight.replace(/\*\*/g, '')
+                                    : card.insight
+                                ),
+                              }}
+                            />
+                          )}
+                        </Fragment>
+                      ) : (
+                        card.insight && (
+                          <div
+                            dangerouslySetInnerHTML={{
+                              __html: marked(
+                                preview ? card.insight.replace(/\*\*/g, '') : card.insight
+                              ),
+                            }}
+                          />
+                        )
+                      )}
+                    </CardContent>
```

Comment:
```suggestion
                    <CardContent>
                      {card.insightElement}
                      {card.insight && (
                        <div
                          dangerouslySetInnerHTML={{
                            __html: marked(
                              preview ? card.insight.replace(/\*\*/g, '') : card.insight
                            ),
                          }}
                        />
                      )}
                    </CardContent>
```
Would this be much cleaner?